android/engine/src/main/java/org/smartregister/fhircore/engine/util/extension/FhirEngineExtension.kt

```diff
@@ -52,10 +52,12 @@ suspend fun FhirEngine.loadLibraryAtPath(fhirOperator: FhirOperator, path: Strin
     runCatching { get<Library>(IdType(path).idPart) }.getOrNull()
       ?: search<Library> { filter(Library.URL, { value = path }) }.map { it.resource }.firstOrNull()
 
-  library?.let {
+  // TODO Fix refactored implementation on fhirOperator.loadLib()
```

Comment:
This should come from the knowledge manager now. Have we added libs there already?

Reply:
We included the knowledger dependency do we need any others? Feel free to add any other that's relevant for the knowledge manager to work.